src/thor/triplegbuilder.cc

```diff
@@ -416,7 +416,7 @@ void RemovePathEdges(valhalla::Location* location, const GraphId& edge_id) {
                             return e.graph_id() == edge_id;
                           });
   if (pos == location->path_edges().end()) {
-    location->mutable_path_edges()->Clear();
+    location->mutable_path_edges()->Clear(); // should never happen
```

Comment:
i kept this here though as the comment says it should never happen. over in #3274 we will change this to a `throw`